### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ElevenTom**

### CODE REVIEW

File: _posts/en/2017-11-08-git-bisect.md [hunk 1]
```diff
@@ -14,14 +14,13 @@ tags:
 cover: /assets/2017-10-26-debugging-with-git/branching-illustration.png
 ---
 
-When deploying new features to a production environment, risks of regression exist. Especially in big applications with a lot of non-tested code.
-We are never safe from negative behaviour, or impacts that need our immediate attention. 
+When deploying new features to a production environment, risks of regression exist. Especially in big applications with a lot of untested code.vWe are never safe from negative behaviour, or impacts that need our immediate attention.
```
Comment (GusTTShowbiz) [inline _posts/en/2017-11-08-git-bisect.md]: typo ici `code.vWe`

**Comment by ElevenWilson** [comment]
8 words might be misspelled, please check them: functionalities, jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, vWe, Yay

File: _posts/en/2017-11-08-git-bisect.md [hunk 2]
```diff
@@ -63,7 +62,7 @@ Once this is done, we can notice that `bisect` changed our position in the commi
 Bisecting: 2 revisions left to test after this (roughly 2 steps)
 [99f3fa1b86489dd9d6f30368d5b5321e04a955df] worked on real-time suggestion when encounting problems in space
 ```
-Now on, we must check if the bug is still here. Unfortunately yes in this case! 
+Now we must check if the bug is still here. Unfortunately yes in this case!
```
Comment (GusTTShowbiz) [inline _posts/en/2017-11-08-git-bisect.md]: Now,

**Comment by ElevenWilson** [comment]
7 words might be misspelled, please check them: functionalities, jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, Yay

**Pull request merged**